libgraph/include/katana/SharedMemSys.h

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
This needs documentation updates. We need to document what parts of the runtime are ending up in what library. Especially for those of us who got used to the old arrangement.